src/server/routes.ts

```diff
@@ -1,9 +1,27 @@
 import express from "express";
+import { fetchGitHubSearchCodeWithUser } from "./gitHubSearch";
 
 const router = express.Router();
 
-router.get("/search", (_req, res) => {
-  res.status(404).send("Hello World");
+router.get("/search", async (_req, res, next) => {
+  try {
+    const { query } = _req;
+    if (!query.code) {
+      res.status(400).send("Code required");
+      return;
+    }
+    if (!query.user) {
+      res.status(400).send("User required");
+      return;
+    }
+    if (typeof query.code !== "string" || typeof query.user !== "string") {
```

Comment:
Good job. `query.code` could be an array if you add it multiple times to your request like:
`/api/search?code=abc&code=123`